app/src/services/redisSessionStorage.js

```diff
@@ -41,11 +41,11 @@ export default class RedisSessionStorage implements SessionStorageInterface {
     return new Promise(function(resolve) {
       // Returns the value associated with field in the hash stored at token.
       // @see https://redis.io/commands/hget
-      client.hget("hash", token, function(err, value) {
+      client.hget("sessions", token, function(err, value) {
```

Comment:
we should use a single constant, instead of a string, making a typo in the name of the constant will trigger an error in the type checker, making a typo in the string will not